hw09-jdbc/src/main/java/ru/otus/hw09/service/DbExecutor.java

```diff
@@ -0,0 +1,17 @@
+package ru.otus.hw09.service;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public interface DbExecutor<T> {
+
+     boolean create(Connection connection, Class<?> clazz);
```

Comment:
```suggestion
     boolean createTable(Connection connection, Class<?> clazz);
```

Я бы уточнил, что это создание таблицы.
Просто `create` может запутать.